### DISCUSSION TITLE
When Will Android 15 Support Be Available on GloDroid?

### DISCUSSION THREAD

**Discussion by indraja** [GloDroid/glodroid_manifest#246] [Q&A]
Hi GloDroid Community,

We’re currently working on a project that requires Android 15 support, and I was wondering if there is an estimated timeline for when Android 15 support will be available on GloDroid.

Are there any plans or ongoing efforts to integrate Android 15?
If so, is there a branch or preview release I could follow?
Any updates or guidance would be greatly appreciated!

Thanks,
Indraja Machani

**Comment by rsglobal** [comment]
Hi @indraja ,

Which platform/device are you interested in?

**Comment by castenaso** [comment]
glo droid 2.0 it is published in 2023, is this project abandoned?

**Reply by castenaso** [reply]
sorry, to ask in this manner, im sitted on sofa and i wait the work of rsglobal, im foreigner and my bad english sound always rude, i dont mean

**Comment by indraja** [comment]
@rsglobal Thank you for your response. We are currently working on Raspberry Pi 4.
Appreciate the Android 15 repository, thanks for creating it!

I’m now interested to know: **When will Android 16 support be available on GloDroid for RPI-4?**
